### DISCUSSION TITLE
Downgrade

### DISCUSSION THREAD

**Discussion by Sam-hacks1** [3kh0/ext-remover#1386] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
What version should I downgrade to, if I want to unblock my chomebook?

**Comment by S-PScripts** [comment]
that really depends on the exploits you want to have

**Reply by S-PScripts** [reply]
i'd recommend versions with incognito, ltbeef or ltmeat (the latter two are versions with javascript: in bookmarklets)
